### PR TITLE
Data Dictionary V1.23

### PR DESCRIPTION
**Pull request by lindaxiang**
**Update**:
1. Added two optional fields in radiation schema to denote radiation boost
2. Allow negative interval in specimen to ingest the data from Monstar

Both dictionary changes have been tested successfully in QA.

### CODE REVIEW

File: references/validationFunctions/radiation/radiationBoost.js [hunk 1]
```diff
@@ -0,0 +1,47 @@
+/*
+ * Copyright (c) 2022 The Ontario Institute for Cancer Research. All rights reserved
```
Comment (justincorrigible) [inline references/validationFunctions/radiation/radiationBoost.js:2]:
```suggestion
 * Copyright (c) 2024 The Ontario Institute for Cancer Research. All rights reserved
```

File: tests/radiation/radiationBoost.test.js [hunk 2]
```diff
@@ -0,0 +1,121 @@
+/*
+ * Copyright (c) 2022 The Ontario Institute for Cancer Research. All rights reserved
```
Comment (justincorrigible) [inline tests/radiation/radiationBoost.test.js:2]:
```suggestion
 * Copyright (c) 2024 The Ontario Institute for Cancer Research. All rights reserved
```

File: tests/radiation/radiationBoost.test.js [hunk 3]
```diff
@@ -0,0 +1,121 @@
...
+
+const myUnitTests = {
+    "reference_radiation_treatment_id": [
+        [
+            'reference_radiation_treatment_id submitted when radiation boost was given.',
+            true,
+            loadObjects(radiation,
+                {   
+                    "radiation_boost": "yes",
+                    "reference_radiation_treatment_id": "tr2"
```
Comment (justincorrigible) [inline tests/radiation/radiationBoost.test.js:36]: Will there ever be a case when treatment ID is "unknown" or other generic values? if so, is that valid?

**Review by justincorrigible — APPROVED** [review]
Looks good, left a couple suggestions and a question, that all can be ignored.

**Comment by lindaxiang** [comment]
If you are DCC admin, you can use the following test files to test the dictionary updates:
QA platform: https://platform-ui.argo-qa.cumulus.genomeinformatics.org/
program: MONSTAR-JP
clinical files:
1. specimen has negative intervals - [specimen_v1.23.txt](https://github.com/user-attachments/files/16663243/specimen_v1.23.txt)
2. treatment - [treatment_v1.23.txt](https://github.com/user-attachments/files/16663246/treatment_v1.23.txt)
3. radiation has two optional fields to denote radiation boost for one donor - [radiation_dictionary_v1.23.txt](https://github.com/user-attachments/files/16663257/radiation_dictionary_v1.23.txt)

File: tests/radiation/radiationBoost.test.js (hunk 3 above)
Comment (lindaxiang) [reply tests/radiation/radiationBoost.test.js:36]: @justincorrigible you pointed out a valid use case. However based on the current validation rule, if indicating `radiation_boost` as `yes` while treatment ID is "Unknown", the program will need to apply exception on field `reference_radiation_treatment_id`.

**Review by justincorrigible — APPROVED** [review]
Ship it! :shipit:

**Pull request merged**